src/Chapter1.hs

```diff
@@ -620,7 +623,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = tensDigit n + onesDigit n
+  where
+    onesDigit :: Int -> Int
+    onesDigit d =  mod (abs d) 10
+    tensDigit :: Int -> Int
+    tensDigit d = onesDigit (div (abs d) 10)
```

Comment:
Beautiful!

One hint to make your solution a bit shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!

So you could write it this way:

```haskell
(first, second) = divMod (abs d) 10
```

You can see how we could pattern match on the pair 🙂 

And also `onesDigit` function, that you use in `tensDigit` is the same as `lastDigit` from the previous task 🙂